src/Donation/DonationRequestUtils.php

```diff
@@ -103,7 +103,7 @@ public function createRetryPayload(Donation $donation, Request $request): array
 
     public function createCallbackStatus(Donation $donation): array
     {
-        $code = self::PAYBOX_STATUSES[$donation->getPayboxResultCode()] ?? self::PAYBOX_UNKNOWN;
+        $code = self::PAYBOX_STATUSES[$donation->getPayboxResultCode()] ?? 'error';
```

Comment:
I thought about that, was not sure of the benefit of a new constant, I guess we should go for `self::PAYBOX_STATUSES[self::PAYBOX_UNKNOWN]` then.